projects/vdk-plugins/vdk-ipython/setup.py

```diff
@@ -16,7 +16,6 @@
     long_description_content_type="text/markdown",
     install_requires=["vdk-core", "iPython"],
     package_dir={"": "src"},
-    packages=setuptools.find_namespace_packages(where="src"),
```

Comment:
I don't understand the fix. Removing it seems like bad idea. We keep sources in "src" folder so we use setuptools.find_namespace_packages  to automatically find all the Python packages within the directory

Reply:
Oh, that's not following the plugin templates. 

The source code should be in in "src" directory in package starting with vdk.plugin.<plugin-name>.... 

That's the structure of all other plugins . It's the one setup if a plugin is used using the cookiecutter - https://github.com/tozka/cookiecutter-vdk-plugin/tree/main

But I don't get the problems? Which files are not included? And why do we need them?

Reply:
vdk_ipython.py is at the root of the source folder. It is not in a package so if that line is in the setup.py then this file is dropped. 

@duyguHsnHsn  do you think we could put the plugin in a package or would it make it less usable for customers do you think?

Reply:
I think I got the problem.

 `find_namespace_packages` doesn't include files that are not in a directory (aka package), and those that are at the root level specified are ignored. I find this a bit strange to be honest. And slightly problematic. 
 
 
The current approach you are doing would also have a namespace/package. If you unzip the tar you will see the file is inside a "src" folder . So a user would need to do 
```
from src import vdk_ipython
```

you can test it yourself by installing `pip install dist/vdk-ipython-0.1.0.tar.gz` 

The test pass because they assume "src" structure is kept and set src on the PythonPath and ignore setup.py